src/UI Component/Adminpanel/Activities/loanapproval.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+
+const LoanApprovalPage = () => {
+  // Dummy data for loan applications
+  const loanApplications = [
```

Comment:
If this is dummy data, move this and any other dummy data to an util.js or constant file, import it and reuse it as required. 

This dummy data will be re-created everytime this component renders. That is not good for performance